server/src/main/java/org/apache/kafka/server/config/ShareCoordinatorConfig.java

```diff
@@ -71,6 +72,10 @@ public class ShareCoordinatorConfig {
     public static final int APPEND_LINGER_MS_DEFAULT = 10;
     public static final String APPEND_LINGER_MS_DOC = "The duration in milliseconds that the share coordinator will wait for writes to accumulate before flushing them to disk.";
 
+    public static final String STATE_TOPIC_PRUNE_INTERVAL = "share.coordinator.state.topic.prune.interval.ms";
```

Comment:
This should be `STATE_TOPIC_PRUNE_INTERVAL_MS_CONFIG` I think. All three of these should have `_MS_` as part of the name.